site/src/components/Resources/Resources.tsx

```diff
@@ -51,129 +53,152 @@ export const Resources: FC<React.PropsWithChildren<ResourcesProps>> = ({
-                                  appIcon={app.icon}
-                                  appName={app.name}
-                                  userName={workspace.owner_name}
+                        </TableCell>
+                        <TableCell>
+                          <div className={styles.accessLinks}>
+                            {canUpdateWorkspace && agent.status === "connected" && (
+                              <>
+                                <SSHButton workspaceName={workspace.name} agentName={agent.name} />
+                                <TerminalLink
                                   workspaceName={workspace.name}
                                   agentName={agent.name}
+                                  userName={workspace.owner_name}
                                 />
-                              ))}
-                            </>
-                          )}
-                          {canUpdateWorkspace && agent.status === "connecting" && (
-                            <>
-                              <Skeleton width={80} height={60} />
-                              <Skeleton width={120} height={60} />
-                            </>
-                          )}
-                        </div>
-                      </TableCell>
-                    </TableRow>
-                  )
-                })
-              })}
-            </TableBody>
-          </Table>
-        </TableContainer>
+                                {agent.apps.map((app) => (
+                                  <AppLink
+                                    key={app.name}
+                                    appIcon={app.icon}
+                                    appName={app.name}
+                                    userName={workspace.owner_name}
+                                    workspaceName={workspace.name}
+                                    agentName={agent.name}
+                                  />
+                                ))}
+                              </>
+                            )}
+                            {canUpdateWorkspace && agent.status === "connecting" && (
+                              <>
+                                <Skeleton width={80} height={60} />
+                                <Skeleton width={120} height={60} />
+                              </>
+                            )}
+                          </div>
+                        </TableCell>
+                      </TableRow>
+                    )
+                  })
+                })}
+              </TableBody>
+            </Table>
+          </TableContainer>
+        )}
+      </div>
+
+      {hasHideResources && (
+        <div className={styles.buttonWrapper}>
+          <Button
+            className={styles.showMoreButton}
+            variant="outlined"
+            size="small"
+            onClick={() => setShouldDisplayHideResources((v) => !v)}
+          >
+            Show hidden resources{" "}
```

Comment:
Thoughts on changing this text when the hidden resources are showing?  My reasoning is that the button will say "show resources" but clicking on it actually hides them.  Maybe we can toggle between `show hidden resources` and `hide resources`?

Reply:
Makes sense!